physical/raft/raft_autopilot.go

```diff
@@ -483,8 +497,8 @@ func (b *RaftBackend) startFollowerHeartbeatTracker() {
 					state.IsDead.Store(true)
 				}
 			}
-			b.followerStates.l.RUnlock()
 		}
+		b.followerStates.l.RUnlock()
```

Comment:
good catch!

Reply:
Oh, the RUnlock wasn't in the loop :-)
:phew:

Reply:
Not that it matters much, but the reason for having the config conditionals before the loop was to avoid grabbing the read lock on follower states. FYI so you can make a known decision.

Reply:
Yup, I assumed that.  But if we're going to iterate over the followers all the time now we need the lock regardless.